filters/imagefilter.go

```diff
@@ -101,3 +103,10 @@ func transformImage(out *io.PipeWriter, image *bimg.Image, opts *bimg.Options) e
 
 	return err
 }
+
+func applyDefaults(o *bimg.Options) *bimg.Options {
+	if o.Quality == 0 {
```

Comment:
I guess you should oppen an issue in the `bimg` library as well. Croping for example, should not reduce the image quality.